### ISSUE TITLE
Add option to generate the source files not in temporary dir

### ISSUE THREAD

**Issue by enricosada** [jet/falanx#58]
Add option to generate the source files not in the temporary `obj` dir, so can be committed in the repo, to make it easier the diff and version.

Maintain the up to date check at build time.

**Comment by 7sharp9** [comment]
A pinned version of Falanx will generate the same code for the same pinned version of a proto file.

**Comment by enricosada** [comment]
implemented in #98 

by default, now it's generated alongside the .proto file, so `my.proto` -> `my.proto.fs` (so it's easy to gitignore as `.proto.fs`)

Is possibile to specify the output path as

```xml
<ProtoFile Include="my.proto">
   <OutputPath>to\another\dir\andfile.fs</OutputPath>
</ProtoFile>
```
